content/lua/concepts/variables/variables.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Variables' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Variables are used to store data that can be referenced throughout the code.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Variables'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+---
+
+Variables are used to store data that can be referenced throughout the code.
+
+Lua has two main types of variables: global variables and local variables. For more information on global and local variables, take a look at the documentation on [Scope](https://www.codecademy.com/resources/docs/lua/scope).
```

Comment:
```suggestion
Lua has two main types of variables: global variables, which can be used anywhere in a program, and local variables, which can only be referenced in the block where they're defined.
```

Removing reference to scope since we didn't get to it in the internship.